.circleci/config.yml

```diff
@@ -10,11 +10,21 @@ jobs:
       - restore_cache:
           keys:
             - functions-dependencies-{{ checksum "functions/package.json" }}
+            - client-dependencies-{{ checksum "package.json" }}
```

Comment:
Is the goal here to retrieve two different caches? If so I believe you should define two reestore_cache blocks, one for `functions-dependencies` and the other for `client-dependencies`. Otherwise, this block will retrieve the cache which key matches the listed prefixes, in the given order, i.e., either `functions-dependencies-*` or `client-dependencies-*`, but not both.

Reply:
Right, based on the document.
List of cache keys to lookup for a cache to restore. Only the first existing key will be restored.